### DISCUSSION TITLE
Does Gcam-Services-Provider work with other camera apps?

### DISCUSSION THREAD

**Discussion by rparkins999** [lukaspieper/Gcam-Services-Provider#62] [Q&A] [answered]
The camera app on my new phone (GalaxyS21) won't get location for its photos without Google Play Services enabled. Does Gcam-Services-Provider only work with Gcam, or will it fake the APis of Google Play Services for other camera apps?

**Comment by lukaspieper** [comment, marked as answer]
Hi,

I'm sorry, but this project is solely focused on Google Camera.
